docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -82,18 +87,17 @@ Thus [`cy.get`](/api/commands/get) and [`cy.find()`](/api/commands/find) have to
 be smarter and expect the application to potentially update. `cy.get()` queries
 the application's DOM, finds the elements that match the selector, and then
 passes them to `.find('.todo-list li')`. `.find()` locates a new set of
-elements, and tries the assertion that follows (in our case
-`should('have.length', 1)`) against the list of found elements.
-
-- ✅ If the assertion that follows `cy.find()` passes, then the query finishes
-  successfully.
-- 🚨 If the assertion that follows `cy.find()` fails, then Cypress will requery
-  the application's DOM again - starting from the top of the list of chain. Then
-  Cypress will try the assertion against the elements yielded from
-  `cy.get().find()`. If the assertion still fails, Cypress continues retrying
-  until the `cy.find()` timeout is reached.
-
-Retry-ability allows the test to complete each query as soon as the assertion
+elements, and passes them to `.should()`. `.should()` then asserts on the list
+of found elements (in our case, that it has a length of 1).
+
+- ✅ If the assertion passes, then `.should()` finishes successfully.
+- 🚨 If the assertion fails, then Cypress will requery the application's DOM
+  again - starting from the top of the chain of linked queries. It will look for
+  elements that match `.get().find()` again, and re-run the assertion. If the
+  assertion still fails, Cypress continues retrying until the timeout is
```

Comment:
```suggestion
  assertion still fails, Cypress continues retrying the query chain until the timeout is
```

Reply:
That's already been mentioned twice in the previous two sentences, I'm not sure stating it a third time in a new way is necessary.